libs/core/files-manager/components/dettached-during-upload-error-modal/dettached-during-upload-error-modal.component.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { useDispatch, useSelector } from "react-redux"
+import { defineMessages } from "react-intl"
+import {
+  ModalContent,
+  ModalDialog,
+  RoundIconWrapper,
+} from "Core/ui/components/modal-dialog"
+import { ModalSize } from "Core/__deprecated__/renderer/components/core/modal/modal.interface"
+import { intl } from "Core/__deprecated__/renderer/utils/intl"
+import Icon from "Core/__deprecated__/renderer/components/core/icon/icon.component"
+import { IconType } from "Core/__deprecated__/renderer/components/core/icon/icon-type"
+import Text, {
+  TextDisplayStyle,
+} from "Core/__deprecated__/renderer/components/core/text/text.component"
+import { FunctionComponent } from "Core/core/types/function-component.interface"
+import { resetUploadingState } from "Core/files-manager/actions/base.action"
+import { isDettachedDuringUploadErrorSelector } from "Core/files-manager/selectors/is-dettached-during-upload-error.selector"
+
+const messages = defineMessages({
+  detachedDuringUploadErrorModalTitle: {
+    id: "module.filesManager.detachedDuringUploadErrorModalTitle",
+  },
+  detachedDuringUploadErrorModalHeader: {
+    id: "module.filesManager.detachedDuringUploadErrorModalHeader",
+  },
+  detachedDuringUploadErrorModalTextInfo: {
+    id: "module.filesManager.detachedDuringUploadErrorModalTextInfo",
+  },
+  detachedDuringUploadErrorModalActionButton: {
+    id: "module.filesManager.detachedDuringUploadErrorModalActionButton",
+  },
+})
+
+const DuplicatedFilesDetailText = styled(Text)`
+  margin-top: 0.8rem;
+`
+
+const DetachedDuringUploadErrorModal: FunctionComponent = ({ ...props }) => {
+  const dispatch = useDispatch()
+  const open = useSelector(isDettachedDuringUploadErrorSelector)
+
+  return (
+    <ModalDialog
```

Comment:
You may consider using of modal components for generic devices

Reply:
You are right, I'm going to do this during the alarm sounds implementation.